src/coredata.cs

```diff
@@ -896,85 +1018,123 @@ public interface NSManagedObjectChangeEventArgs {
 	[DisableDefaultCtor]
 	public interface NSManagedObjectID : NSCopying {
 
-		[Export ("entity")]
+		[Export ("entity", ArgumentSemantic.Strong)]
 		NSEntityDescription Entity { get; }
 
-		[Export ("persistentStore")]
+		[NullAllowed, Export ("persistentStore", ArgumentSemantic.Weak)]
 		NSPersistentStore PersistentStore { get; }
 
-		[Export ("isTemporaryID")]
-		bool IsTemporaryID { get; }
+		[Export ("temporaryID")]
+		bool IsTemporaryID { [Bind ("isTemporaryID")] get; }
 
 		[Export ("URIRepresentation")]
 		NSUrl URIRepresentation { get; }
 
 	}
 
 	[BaseType (typeof (NSObject))]
-	public interface NSManagedObjectModel : NSCoding, NSCopying {
+	public interface NSManagedObjectModel : NSCoding, NSCopying, INSFastEnumeration {
```

Comment:
it would be `NSFastEnumeration` and we do not support it (at least not this way) right now